dist/lib/Client.js

```diff
@@ -45,7 +45,12 @@ const {
   SMART_KEY,
   patientCompartment,
   fhirVersions
-} = require("./settings");
+} = require("./settings"); // eslint-disable-next-line no-undef
+
+
+const {
```

Comment:
I have some issues with this statement:
1. TypeScript is unable to understand `global.FHIRCLIENT_PURE`. The same applies to the other files where it is used. So far my fix is to add this in `types.d.ts`:
    ```ts
    declare module NodeJS {
        interface Global {
            FHIRCLIENT_PURE?: Boolean 
        }
    }
    ```
    I'm open to suggestions if you know another way to fix this.
2. In browsers, `Response` is a global object so the second part should be `window` instead of `window.fetch`. In other words:
```js
const { Response } = global.FHIRCLIENT_PURE ? window : require("cross-fetch");
```

Reply:
1. I think it depends on if we want this variable to be available outside the library? It's only a webpack build detail, so if this were converted to TypeScript code, I would probably write it `(global as any).FHIRCLIENT_PURE`.

Reply:
I was talking about the TS validation service that runs in my VS Code editor, even on JS files. Anyway, I found that simple `// @ts-ignore` comment is good enough to fix that warning.